### PR TITLE
Update bootstrap farmers market offering

### LINKED ISSUE #474: Update Bootstrap script for offerings product context
- Limit dropdown to 15 items for now (eventually we’ll need to make this configurable or something) 
- Change the offering product title to “King Farmers Market 1 to 1 Voucher Match”
- Edit bootstrap script so memo is the same as ledger name, change automation implementations to default memo to the ledger contribution name
- Each offering product (“first time buyer” v “returning buyer”) should have a unique photo – (each image needs to have Es and En in it, Malena putting into the slack)
- Translate to Spanish

Todo:

- [ ] Update above in production (safely)
- [x] Add new product images for both farmers market offerings
- [x] Update BS lime rate `unit_amount` to 0 (free) and change rate context in UI (also update in production)

### PR DESCRIPTION
**Pull request by DeeTheDev**
Fixes #474 

- Update king farmers market offering details (check issue)
- Update Funding transaction match automation _subsidy memo_ to the  _contribution text when it is not present
  - Update Rspec tests

### CODE REVIEW

File: lib/suma/automation_trigger/funding_transaction_match.rb [hunk 1]
```diff
@@ -41,7 +42,7 @@ def run
         originating_ledger: Suma::Payment::Account.lookup_platform_vendor_service_category_ledger(vsc),
         receiving_ledger: ledger,
         associated_vendor_service_category: vsc,
-        memo: Suma::TranslatedText.create(**params.fetch(:subsidy_memo)),
+        memo: contribution_text,
```
Comment (rgalanakis) [inline lib/suma/automation_trigger/funding_transaction_match.rb:45]: Allow memo to be optional, and default to the contribution text. There will be cases where we want a separate memo.

File: lib/suma/tasks/bootstrap.rb [hunk 2]
```diff
@@ -390,14 +390,14 @@ def setup_farmers_market(market_name:, market_address:, logo:, hero:, offering_p
...
-      name_es: "$2 en Cupones de #{market_name}",
+      name_en: "#{market_name} 1 to 1 Voucher Match",
+      name_es: "#{market_name} 1 a 1 de Cupones Igualados",
       description_en: "The suma voucher is a food special where suma works with you to buy down the price of vouchers for fresh and packaged food at #{market_name}. suma will match you 1:1 up to a $30 total (you load $15, suma matches with $15). You cannot use these vouchers for alcohol or hot prepared foods.",
       description_es: "El cupón de suma es un especial de alimentos en el que suma trabaja con usted para reducir el precio de los cupones para alimentos frescos y envasados en #{market_name}. suma te igualara 1:1 hasta un total de $30 (tu agregas $15, suma agrega $15 de créditos). No puede utilizar estos cupones para bebidas alcohólicas o comidas preparadas calientes.",
       our_cost: Money.new(200),
       vendor_service_categories: [fm2023_match_category],
       logo:,
-      max_quantity_per_order: 500,
+      max_quantity_per_order: 15,
```
Comment (rgalanakis) [inline lib/suma/tasks/bootstrap.rb:400]: This needs to be a UI change, not a data change. This means we can't purchase more than 15 for an offering/user.

File: spec/suma/automation_trigger_spec.rb [hunk 3]
```diff
@@ -164,6 +164,16 @@
       )
     end
 
+    it "defaults memo as contribution text if nil", lang: :es do
+      at.update(parameter: at.parameter.merge(subsidy_memo: nil))
```
[the remark this answers is not in the document]
Comment (DeeTheDev) [reply spec/suma/automation_trigger_spec.rb:168]: deleting the key errors null violation: 
```
Sequel::NotNullConstraintViolation: PG::NotNullViolation: ERROR:  null value in column "parameter" of relation "automation_triggers" violates not-null constraint
DETAIL:  Failing row contains (107, 2023-08-03 01:57:00.920067+00, 2023-08-03 01:57:00.922014+00, Praesentium aspernatur libero nulla., ["2023-07-09 01:57:00.919826+00","2023-08-11 01:57:00.919864+00"..., *, Suma::AutomationTrigger::FundingTransactionMatch, null).
```

File: webapp/src/components/FoodCartWidget.js [hunk 4]
```diff
@@ -77,6 +77,10 @@ export default function FoodCartWidget({ product, size, onQuantityChange }) {
     );
   }
 
+  const maxQuantity =
```
[the remark this answers is not in the document]
Comment (DeeTheDev) [reply webapp/src/components/FoodCartWidget.js:80]: Very clever I like it.

File: spec/suma/automation_trigger_spec.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (DeeTheDev) [reply spec/suma/automation_trigger_spec.rb:168]: Will give it a try, was deleting the whole parameter I believe.

**Pull request merged**